src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -0,0 +1,30 @@
+/*xml
+<module name="Checker">
+  <module name="SuppressionSingleFilter">
+    <property name="checks" value="JavadocStyleCheck"/>
+    <property name="files" value="Example1.java"/>
+    <property name="lines" value="1,5-100"/>
+  </module>
+  <module name="SuppressionSingleFilter">
+    <property name="checks" value="MagicNumberCheck"/>
+    <property name="files" value="Example1.java"/>
+    <property name="lines" value="1,5-100"/>
+  </module>
+  <module name="SuppressionSingleFilter">
+    <property name="message" value="Missing a Javadoc comment"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.filters.suppressionsinglefilter;
+// xdoc section -- start
+public class Example1 {
+
+  // This method is missing a Javadoc comment
+  // which should trigger JavadocStyleCheck violation
+  public void exampleMethod() {
+    int value = 100;
+    // This should trigger MagicNumberCheck violation, as 100 is a magic number
```

Comment:
please use this approach for marking "expected without filter" violation.
https://github.com/checkstyle/checkstyle/pull/16282/files#diff-689baf75b205d21519c8ec41be0ebadfc653657287e46b5fb4cd220a5532963eR25

please review this PR and do code  updates in same way.